src/services/API/Event.js

```diff
@@ -10,4 +14,12 @@ export default {
         return response.data;
       });
   },
+  /**
+   *
+   * @param {Number} eventId - The ID of the event that needs to be patched
+   * @param {Object} eventData - The event will be patched with this data
```

Comment:
```suggestion
   * @param {Object} eventData - The event will be updated with this data
```

Reply:
Updated it according to the method `createEvent` above.
Also, I think let's stick with `payload` term. It is generic enough and it makes sense in every API call.